### DISCUSSION TITLE
Zend Guard Loader 6 and Centreon 2.8.32

### DISCUSSION THREAD

**Discussion by hgallacier** [centreon-archive/centreon-archived#9726] [General]
Hello,
I have a server with Centos 7 and PHP 5.4.16.
I installed the rpm of Zend Guard Loader in the version 6.0 and Zendid.

`
PHP 5.4.16 (cli) (built: Apr  1 2020 04:07:17)
Copyright (c) 1997-2013 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2013 Zend Technologies
    with Zend Guard Loader v3.3, Copyright (c) 1998-2013, by Zend Technologies

zendid-1.0-1.el7.centos.x86_64
php-zend-guard-loader-6.0.0-4.el7.centos.x86_64
`

I can display the page of Centreon MBI. I can generate a report but it is empty.

Moreover, the download button in the website is not displayed.

Could you hel me to understand this issue ?

Thank you !

**Comment by lpinsivy** [comment]
Dear,

Centreon 3.4 (Centreon Web 2.8.x) is not more supported by Centreon, please see our [products lifecycle policy](https://docs.centreon.com/current/en/releases/lifecycle.html#centreon-oss-34)

You can contact the Centreon support team for assistance.

Regards,
